apis/org_handler_test.go

```diff
@@ -0,0 +1,154 @@
+package apis_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"code.cloudfoundry.org/cf-k8s-api/apis"
+	"code.cloudfoundry.org/cf-k8s-api/apis/fake"
+	"code.cloudfoundry.org/cf-k8s-api/repositories"
+	. "github.com/onsi/gomega"
+	"github.com/sclevine/spec"
+)
+
+var _ = SuiteDescribe("listing orgs", func(t *testing.T, when spec.G, it spec.S) {
```

Comment:
So far we've only used this pattern for tests that require env-test, like the repository tests. It's necessary there because we need something equivalent to "BeforeSuite". This doesn't seem necessary here and it would be better to just use `spec.Run` directly like in the other handler tests.

Reply:
Sure. I've aligned it with the other handler tests.